apis/apps/v1alpha1/type.go

```diff
@@ -321,6 +321,33 @@ const (
 	UnavailablePhase Phase = "Unavailable"
 )
 
+// StateMode represents the state mode of a CR object.
+//
+// +enum
+// +kubebuilder:validation:Enum={Running,Stopped}
+type StateMode string
+
+const (
+	// StateModeRunning indicates that the object is in a running state.
+	StateModeRunning StateMode = "Running"
+
+	// StateModeStopped indicates that the object is in a stopped state.
+	StateModeStopped StateMode = "Stopped"
```

Comment:
on/off ?

Reply:
When a user sets the state to 'Running', the phase of the Cluster might not necessarily transition to 'Running'. Instead, it could remain in other states such as 'Creating' or 'Pending'. This discrepancy between the user-set state and the actual phase can lead to confusion.

Reply:
What the users can expect from a `Running` object? The DB is serving well?

Reply:
Rename the enum state to a stop flag.